src/Microsoft.AspNetCore.JsonPatch/project.json

```diff
@@ -21,22 +21,17 @@
     ]
   },
   "dependencies": {
+    "Microsoft.Extensions.ClosedGenericMatcher.Sources": {
+      "type": "build",
+      "version": "1.1.0-*"
+    },
     "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "net451": {},
+    "netstandard1.3": {
```

Comment:
Changing the supported frameworks remains a breaking change.

@kichalla you mentioned this had something to do w/ `ClosedGenericMatcher`. Could the requirements of that package be reduced?

@Eilon thoughts?